### PR TITLE
adminer,minio stacks

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- [x] adds adminer stack
- [x] adds minio stack (deploys for both single and distributed mode)

## Related issue number

links osparc-simcore#575

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: services/maintenance/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# portainer stack
```
Comment (pcrespov) [inline services/maintenance/README.md]: The portainer stack under maintenance??

File: services/minio/.env [hunk 2]
```diff
@@ -0,0 +1,6 @@
+MINIO_ACCESS_KEY=12345678
```
Comment (pcrespov) [inline services/minio/.env]: rename as `.env-defaults` and autogenerate `.env` if it does not exist (as in simcore Makefile)?

File: services/minio/docker-compose.yaml.j2 [hunk 3]
```diff
@@ -0,0 +1,65 @@
+version: '3.7'
```
Comment (pcrespov) [inline services/minio/docker-compose.yaml.j2]: VERY nice!

File: services/maintenance/README.md (hunk 1 above)
Comment (sanderegg) [reply services/maintenance/README.md]: well, I did not touch this part, remember we did this together when copying the maintenance stack out of simcore. Shall I take care of it?

File: services/minio/.env (hunk 2 above)
Comment (sanderegg) [reply services/minio/.env]: actually was wondering why we have it like this?

Comment (sanderegg) [reply services/minio/.env]: will be fixed in next PR

**Pull request merged**